### PR TITLE
cherry-pick PRs submitted by robots can be verified.

### PR DESCRIPTION
**Pull request by ty-dc**
#### What type of PR is this?

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by weizhoublue** [comment]
any update ?

**Review by Icarus9913 — DISMISSED** [review]

**Review by Icarus9913 — APPROVED** [review]

File: .github/workflows/auto-cherrypick.yaml [hunk 1]
```diff
@@ -107,7 +114,7 @@ jobs:
           echo "============ begin to cherry pick ============ "
           FINAL_FAILURE=false
           git branch
-          git config  user.email "robot@example.com"
+          git config  user.email "tao.yang@daocloud.io"
```
Comment (cyclinder) [inline .github/workflows/auto-cherrypick.yaml:117]: 这是用你的邮箱签名是吗

File: .github/workflows/auto-cherrypick.yaml [hunk 2]
```diff
@@ -184,7 +191,7 @@ jobs:
               if [ "$UPDATE" == "true" ] ; then
                   echo "succeeded to cherry pick to branch $BRANCH "
                   # create a pr
-                  git commit -s --amend --no-edit
+                  git commit -s -S --amend --no-edit
```
Comment (cyclinder) [inline .github/workflows/auto-cherrypick.yaml:194]:
```suggestion
                  git commit -S --amend --no-edit
```
？

**Review by cyclinder — APPROVED** [review]

File: .github/workflows/auto-cherrypick.yaml (hunk 2 above)
Comment (ty-dc) [reply .github/workflows/auto-cherrypick.yaml:194]: -s, --signoff         add a Signed-off-by trailer
-S, --gpg-sign[=<key-id>]  GPG sign commit

**Pull request merged**